faunadb-common/src/main/java/com/faunadb/common/RefAwareHttpClient.java

```diff
@@ -29,13 +28,13 @@ public void close() {
       if (refCount.decrementAndGet() < INITIAL_REF_COUNT && !delegate.isClosed()) {
         delegate.close();
       }
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
I removed because the delegate is not throwing this exception anymore, but I was afraid to break any code because this try/catch capture this exception and wrap it into an IOError